tutorials/asr/ASR_with_NeMo.ipynb

```diff
@@ -0,0 +1,920 @@
+        "Note that this is a stage during which we also tell the model where our training and validation manifests are."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Cbf0fsMK09lk",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "from omegaconf import DictConfig\n",
+        "params['model']['train_ds']['manifest_filepath']=train_manifest\n",
+        "params['model']['validation_ds']['manifest_filepath']=test_manifest\n",
+        "first_asr_model = nemo_asr.models.EncDecCTCModel(cfg=DictConfig(params['model']), trainer=trainer)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "inRJsnrz1psq",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Start training!!!\n",
+        "trainer.fit(first_asr_model)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jpYXX-GslR0E",
+        "colab_type": "text"
+      },
+      "source": [
+        "There we go! We've put together a full training pipeline for the model and trained it for 50 epochs. Let's start Tensorboard to see how training went. Note that WER stands for Word Error Rate and the lower it is, the better.\n",
+        "You can definetely imporove this run by playng with hyperparametrers."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "7kdQbpohXnEd",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Speaking of hyper parameters. Here they are:\n",
+        "print(params['model']['optim'])"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "AbigFKUtYgvn",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# To change, for example, learning rate, do:\n",
+        "import copy\n",
+        "new_opt = copy.deepcopy(params['model']['optim'])\n",
+        "new_opt['lr'] = 0.001\n",
+        "quartznet.setup_optimization(optim_config=DictConfig(new_opt))\n",
+        "# And then you can invoke trainer.fit(first_asr_model)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "n_0y3stSXDX_",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "# Let's start Tensorboard to see how our training went.\n",
+        "%load_ext tensorboard\n",
+        "%tensorboard --logdir lightning_logs/"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PSJeNwkTXQjA",
+        "colab_type": "text"
+      },
+      "source": [
```

Comment:
Add section about batch inference w/ WER evaluation?
I think it'd be useful to have a section (applicable to both the from-scratch and pretrained models) detailing how to do both individual sample transcription and batch inference.

Header: Inference
- Subheader: Individual Sample Inference (e.g. what is shown here)
- Subheader: Batch Inference (e.g. how to pass in a validation set and get WER)

Reply:
added inference section